src/charm.py

```diff
@@ -571,6 +563,72 @@ def _gnb_subnet_config_is_valid(self) -> bool:
     def _get_gnb_subnet_config(self) -> Optional[str]:
         return self.model.config.get("gnb-subnet")
 
+    def _get_access_nad_config(self) -> Dict[Any, Any]:
+        return {
+            "cniVersion": "0.3.1",
+            "ipam": {
+                "type": "static",
+                "routes": [
+                    {
+                        "dst": self._get_gnb_subnet_config(),
+                        "gw": self._get_access_network_gateway_ip_config(),
+                    },
+                ],
+                "addresses": [
+                    {
+                        "address": self._get_access_network_ip_config(),
+                    }
+                ],
+            },
+            "capabilities": {"mac": True},
+        }
+
+    def _get_core_nad_config(self) -> Dict[Any, Any]:
+        return {
+            "cniVersion": "0.3.1",
+            "ipam": {
+                "type": "static",
+                "addresses": [
+                    {
+                        "address": self._get_core_network_ip_config(),
+                    }
+                ],
+            },
+            "capabilities": {"mac": True},
+        }
+
+    def _get_core_interface_mtu_config(self) -> Optional[int]:
+        return self.model.config.get("core-interface-mtu-size")  # type: ignore
+
+    def _get_access_interface_mtu_config(self) -> Optional[int]:
+        return self.model.config.get("access-interface-mtu-size")  # type: ignore
+
+    def _access_interface_mtu_size_is_valid(self) -> bool:
+        """Checks whether the access interface MTU size is valid.
+
+        Returns:
+            bool: Whether access interface MTU size is valid
+        """
+        if self._get_access_interface_mtu_config() is None:
```

Comment:
```suggestion
        if not self._get_access_interface_mtu_config():
```

Reply:
We need to use  "if self._get_access_interface_mtu_config() is None". Reason is use may set 0 as value (which is invalid) and following code block return True which should be False. 
 if not self._get_access_interface_mtu_config():
      return True
      
 If config is empty, the following block is fine.
 if self._get_access_interface_mtu_config() is None:
     return True